TestFoundation/TestNSFileManager.swift

```diff
@@ -137,6 +138,26 @@ class TestNSFileManger : XCTestCase {
         }
     }
     
+    func test_setFileAttributes() {
+        let path = "/tmp/testfile"
```

Comment:
using some sort of unique identifier per-run would be nice to have as a common accessor pattern. perhaps `/tmp/<some name identifying the test><uuid>` might be good as a pattern